fixtures/events/event_checkin_fixtures.py

```diff
@@ -98,3 +98,42 @@
         }
     }
 }
+
+event_existing_in_db_checkin_mutation = '''mutation {
```

Comment:
I would suggest going with `check_in_mutation_fot_existing_event_in_db`

Reply:
I thought this was also descriptive enough, but that is also a good one @RonKbS , I'll go with yours.....done @RonKbS